src/patternfly/components/Drawer/drawer-splitter.hbs

```diff
@@ -0,0 +1,13 @@
+<div class="pf-c-drawer__splitter{{#if drawer-splitter--modifier}} {{drawer-splitter--modifier}}{{/if}}{{#if drawer-splitter--IsVertical}} pf-m-vertical{{/if}}"
```

Comment:
@jessiehuff does this need an `aria-label`?

Reply:
I think this is probably ok. When testing in VO, it announces as "horizontal splitter" or "vertical splitter" because of the aria attributes so I think this is probably fine.